src/main/java/org/spongepowered/api/event/achievement/GrantAchievementEvent.java

```diff
@@ -41,6 +43,7 @@
      */
     Achievement getAchievement();
 
+    @ImplementedBy(AbstractMessageEvent.class)
```

Comment:
This shouldn't be needed - just bump `event-gen-core` and `event-impl-gen` to `1.1.0` in `build.gradle`.